lib/fluent/logger/fluent_logger.rb

```diff
@@ -34,6 +34,28 @@ class FluentLogger < LoggerBase
         r / RECONNECT_WAIT_INCR_RATE
       }
 
+      module Severity
+        # Low-level information, mostly for developers.
+        DEBUG = 0
+        # Generic (useful) information about system operation.
+        INFO = 1
+        # A warning.
+        WARN = 2
+        # A handleable error condition.
+        ERROR = 3
+        # An unhandleable error that results in a program crash.
+        FATAL = 4
+        # An unknown message that should always be logged.
+        UNKNOWN = 5
+      end
+      include Severity
+
+      # Logging severity threshold (e.g. <tt>Logger::INFO</tt>).
+      attr_accessor :level
```

Comment:
ruby 2.3.0 supports a level setter with a string or a symbol. It should be nice to have it